psh/tools/common.py

```diff
@@ -38,6 +41,30 @@ def create_testdir(p, dirname):
     psh.assert_cmd(p, f'mkdir {dirname}', '', msg)
 
 
+def assert_mtime(p, datetimes: Dict[str, datetime], dir=''):
+    ''' Asserts that files (keys in datetimes dictionary) have modification time
+    equal to corresponding datetime values with 1 min margin.
+    Year and seconds are not checked. All files shall be present in the `dir` directory. '''
+
+    dir_files = {file.name: file for file in psh.ls(p, dir)}
+
+    for filename, target_datetime in datetimes.items():
+        assert filename in dir_files, f'File {filename} is not present in the passed directory!'
+        date = dir_files[filename].datetime
+
+        # we do not want to compare years and seconds (not printed by ls -l)
+        date = date.replace(year=target_datetime.year)
+        target_datetime = target_datetime.replace(second=0)
```

Comment:
Maybe we just can set `date.second` to `target_datetime.second` like this:
```suggestion
        date = date.replace(year=target_datetime.year, second=target_datetime.second)
```
The ls function sets seconds to zero anyways so we don't need the second line